docs/source/institutional-faq.rst

```diff
@@ -0,0 +1,362 @@
+
+You can start using Dask without any capital expenditure.
+
+
+How do I manage users?
+~~~~~~~~~~~~~~~~~~~~~~
+
+Dask doesn't manage users, you likely have existing systems that do this well.
+In a large institutional setting we assume that you already have a resource
+manager like Yarn (Hadoop), Kubernetes, or PBS/SLURM/SGE/LSF/..., each of which
+have excellent user management capabilities, which are likely preferred by your
+IT department anyway.
+
+Dask is designed to operate with user-level permissions, which means that
+your data science users should be able to ask those systems mentioned above for
+resources, and have their processes tracked accordingly.
+
+However, there are institutions where analyst-level users aren't given direct access to
+the cluster.  This is particularly common in Cloudera/Hortonworks Hadoop/Spark deployments.
+In these cases some level of explicit indirection may be required.  For this, we
+recommend the `Dask Gateway <https://gateway.dask.org>`_, which uses IT-level
+permissions to properly route authenticated users into secure resources.
+
+
+How do I manage software environments?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This depends on your cluster resource manager:
+
+-  Most HPC users use their network file system
+-  Hadoop/Spark/Yarn users package their environment into a tarball and ship it
+   around with HDFS (Dask-Yarn integrates with `Conda Pack <TODO>`_ for this
+   capability)
+-  Kubernetes or Cloud users typically use Docker images
+
+In each case Dask typically integrates with existing processes and technologies
+that are well understood and familiar to the institution.
+
+
+For Technical Leads
+-------------------
+
+Will Dask "just work" on our existing code?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+No, you will need to make modifications,
+but these modifications are usually relatively small.
+Certainly the vast majority of lines of business logic within your institution
+will not have to change, assuming that they are in Python and use tooling like
+Numpy, Pandas and Scikit-Learn.
+
+How well does Dask scale?  What are Dask's limitations?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The largest Dask deployments that we see today are on around 1000 multi-core
+machines, perhaps 20,000 cores in total, but these are rare.
+Most institutional-level problems (1-100 TB) are well solved by deployments of 10-50 nodes.
+
+Technically, the back-of-the-envelope number to keep in mind is that each task
+(an individual Python function call) in Dask has an overhead of around *200
+microseconds*.  So if these tasks take 1 second each, then Dask can saturate
+around 5000 cores before scheduling overhead dominates costs.  As workloads
+reach this limit they are encouraged to use larger chunk sizes to compensate.
+The *vast majority* of institutional users though do not reach this limit.
+For more information you may want to peruse our :doc:`best practices
+<best-practices>`
+
```

Comment:
One question I get.  How does Dask move data around?

Reply:
Send [this](https://github.com/dask/distributed/tree/master/distributed/protocol) around has been helpful.  Perhaps it would be good to list out the serializers [here](https://distributed.dask.org/en/latest/serialization.htmll).